src/java/org/apache/cassandra/index/sai/IndexContext.java

```diff
@@ -416,6 +421,11 @@ public void update(DecoratedKey key, Row oldRow, Row newRow, Memtable memtable,
         target.update(key, oldRow.clustering(), oldValue, newValue, memtable, opGroup);
     }
 
+    public void createMemtable(Memtable memtable)
+    {
+        liveMemtables.put(memtable, MemtableIndex.createIndex(this, memtable));
```

Comment:
Given that we call this on "renewed" at the moment, it seems like we could overwrite a value, but I haven't checked thoroughly. Can we use `computeIfAbsent`?

Reply:
I reverted that change.